docs/kikoff-guide.mdx

```diff
@@ -612,13 +612,12 @@ The 1-Click endpoint is [`POST /1-click`](/api-reference#1-click). Call it to be
 **We recommend that you use the following request body:**
 ```typescript title="POST /1-click Request Body: Recommendation for Kikoff"
 {
-    "phone": string, // 10-digit string with +1 country code at the beginning
-    "email": string,
+    "phone": string // 10-digit string with +1 country code at the beginning
 }
 ```
 
 :::note[Why this request body?]
-- `phone` and `email` allow us to source verified data about the user and send it to you.
```

Comment:
When did we decide to drop `email`?

Reply:
https://verified-inc.slack.com/archives/CMHP6C3MY/p1706057214499859?thread_ts=1706053723.975469&cid=CMHP6C3MY